go.mod

```diff
@@ -5,10 +5,10 @@ go 1.19
 require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/fsnotify/fsnotify v1.6.0
-	github.com/go-logr/logr v1.2.3
-	github.com/onsi/ginkgo/v2 v2.8.4
-	github.com/onsi/gomega v1.27.2
-	github.com/openshift/api v0.0.0-20230228142948-d170fcdc0fa6 // release-4.13
+	github.com/go-logr/logr v1.2.4
+	github.com/onsi/ginkgo/v2 v2.9.1
+	github.com/onsi/gomega v1.27.4
+	github.com/openshift/api v0.0.0-20230503133300-8bbcb7ca7183 // release-4.13
```

Comment:
nit - Commit `8bbcb7ca7183` is not on `release-4.13` branch of the `github.com/openshift/api`. So the comment should be changed to `// release-4.14`.

It can be done in a future PR, no need to restart CI because of that.

Reply:
https://github.com/kubevirt/ssp-operator/pull/600